src/components/Signup/Signup.test.js

```diff
@@ -333,6 +333,7 @@ describe('Signup', () => {
       utm_medium: null,
       utm_term: null,
       gclid: null,
+      utm_content: null
```

Comment:
This paramter should not be null because `location` has the query param `utm_ content`.
you are rendering `signup` with: `const location = { search: 'test', pathname: '/signup/?utm_content=test-utem-content' };`

Reply:
You can do two tests: the **first** with the query param `utm_content` and the **second** without it.